File: rating-data-service/src/main/java/org/kimbs/ratingdataservice/controller/RatingController.java
package org.kimbs.ratingdataservice.controller;

import java.util.Arrays;
import java.util.List;

import org.kimbs.ratingdataservice.models.Rating;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/rating")
public class RatingController {


    @GetMapping("/{movieId}")
    public Rating getRating(@PathVariable("movieId") String movieId) {
        return new Rating(movieId, 4);
    }

    @GetMapping("/users/{userId}")
    public List<Rating> getUserRating(@PathVariable("userId") String userId) {
        return Arrays.asList(
            new Rating("Test001", 4),
            new Rating("Test002", 3),
            new Rating("Test003", 10)
        );
    }
}

File: rating-data-service_1/src/main/kotlin/org/kimbs/ratingdataservice/controller/RatingController.kt
package org.kimbs.ratingdataservice.controller;

import java.util.Arrays;
import java.util.List;

import org.kimbs.ratingdataservice.models.Rating;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/rating")
class RatingController {

    @GetMapping("/{movieId}")
    fun getRating(@PathVariable("movieId") movieId: String): Rating =
        Rating(movieId, 4)

    @GetMapping("/users/{userId}")
    fun getUserRating(@PathVariable("userId") userId: String): ArrayList<Rating> {
        val arrayList = ArrayList<Rating>();
        arrayList.add(Rating("Test001", 4));
        arrayList.add(Rating("Test002", 3));
        arrayList.add(Rating("Test003", 10));
        return arrayList;
    }
        
}